src/rules/de.toml

```
@@ -7,20 +7,22 @@ quote_start_with_letter = true
 needs_punctuation_end = false
 needs_letter_start = true
 needs_uppercase_start = true
-allowed_symbols_regex = "[\u0020-\u007BäöüßÄÖÜ„“‚‘’–\\.]"
+allowed_symbols_regex = "[\u0020A-za-zäöüßÄÖÜ\"„“‚‘’–\\.?!()]"
```

```suggestion
allowed_symbols_regex = "[\u0020A-Za-zäöüßÄÖÜ\"„“‚‘’–\\.?!()]"
```

I'm sorry, I made a mistake when I suggested this. :( Today is not my day somehow..